gradient/api_sdk/serializers/job.py

```diff
@@ -1,9 +1,30 @@
+import json
+
 import marshmallow
 
 from .base import BaseSchema
 from .. import models
 
 
+class JSONField(marshmallow.fields.Field):
+    """Field that serializes to json
+    """
+
+    def _serialize(self, value, attr, obj, **kwargs):
+        if value is None:
+            return None
+
+        return json.dumps(value)
+
+    def _deserialize(self, value, attr, data, **kwargs):
+        try:
+            result = json.loads(value)
+            if not result:
+                return None
+        except error:
+            raise "Could not parse %s json" % attr
```

Comment:
Result never gets returned, not sure if this is actually used or just a placeholder function